packages/puppeteer-core/src/cdp/CDPSession.ts

```diff
@@ -40,12 +41,14 @@ export class CdpCDPSession extends CDPSession {
     targetType: string,
     sessionId: string,
     parentSessionId: string | undefined,
+    rawErrors: boolean,
```

Comment:
Mark as internal

Reply:
the entire constructor is internal here. For connection, I think it is okay to keep it public.